Build failure: papis

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

```sh
$ nix build .#papis
error: Cannot build '/nix/store/l8cvyv4wqqnirkgifr3k4r3bqwr1iy9r-python3.13-papis-0.14.1.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/bn5yqs47ddmjfx2wq0vj3lsg9j3q78ff-python3.13-papis-0.14.1-dist
         /nix/store/c5b4b9anahy0a4pp9icbdwdr2jydm1d3-python3.13-papis-0.14.1
       Last 25 log lines:
       > FAILED tests/commands/test_tag.py::test_tag_remove_from_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_drop_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_drop_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_rename_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_rename_missing_value_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_set_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_set_clean_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_set_force_str_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_to_new_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_to_int_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_str_to_empty_int_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_clean_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_del_duplicates_in_list_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_remove_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_remove_from_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_remove_from_str_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_drop_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_drop_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_rename_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_rename_missing_value_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_batch_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_yaml_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_bibtex_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > = 65 failed, 161 passed, 5 skipped, 1 deselected, 9 xfailed, 1 warning in 14.20s =
       For full logs, run:
         nix log /nix/store/l8cvyv4wqqnirkgifr3k4r3bqwr1iy9r-python3.13-papis-0.14.1.drv
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/310712361

### Relevant log output

```console
error: Cannot build '/nix/store/l8cvyv4wqqnirkgifr3k4r3bqwr1iy9r-python3.13-papis-0.14.1.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/bn5yqs47ddmjfx2wq0vj3lsg9j3q78ff-python3.13-papis-0.14.1-dist
         /nix/store/c5b4b9anahy0a4pp9icbdwdr2jydm1d3-python3.13-papis-0.14.1
       Last 25 log lines:
       > FAILED tests/commands/test_tag.py::test_tag_remove_from_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_drop_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_drop_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_rename_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_tag.py::test_tag_rename_missing_value_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_set_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_set_clean_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_set_force_str_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_to_new_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_to_int_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_str_to_empty_int_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_clean_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_append_del_duplicates_in_list_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_remove_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_remove_from_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_remove_from_str_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_drop_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_drop_missing_key_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_rename_general_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_rename_missing_value_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_batch_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_yaml_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > FAILED tests/commands/test_update.py::test_update_bibtex_cli - TypeError: CliRunner.__init__() got an unexpected keyword argument 'mix_std...
       > = 65 failed, 161 passed, 5 skipped, 1 deselected, 9 xfailed, 1 warning in 14.20s =
       For full logs, run:
         nix log /nix/store/l8cvyv4wqqnirkgifr3k4r3bqwr1iy9r-python3.13-papis-0.14.1.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.52, NixOS, 25.11 (Xantusia), 25.11.20251015.544961d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/rcsmdj802sk3w9l8m0b0bvdy0jiw2wgr-source`

### Notify maintainers

@teto @nico202 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/papis/default.nix
+++ b/pkgs/development/python-modules/papis/default.nix
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch2,
 
   # build-system
   hatchling,
@@ -121,6 +122,14 @@ buildPythonPackage rec {
   disabledTests = [
     # Require network access
     "test_yaml_unicode_dump"
+  ];
+
+  patches = [
+    (fetchpatch2 {
+      name = "fix-support-new-click-in-papisrunner.patch";
+      url = "https://github.com/papis/papis/commit/0e3ffff4bd1b62cdf0a9fdc7f54d6a2e2ab90082.patch?full_index=1";
+      hash = "sha256-KUw5U5izTTWqXHzGWLibtqHWAsVxla6SA8x6SJ07/zU=";
+    })
   ];
 
   meta = {

